app/models/reports/region_entry.rb

```diff
@@ -0,0 +1,16 @@
+module Reports
+  class RegionEntry
```

Comment:
I like the change from `RegionItem` to `RegionEntry` because it didn't feel right for `RegionItem` to be the same as `Item` but with less period specificity. If anything, it left me thinking `Item` should be renamed to `RegionItem` and this renamed to `RegionPeriodItem`.